tests/h/search/query_test.py

```diff
@@ -141,6 +141,50 @@ def search(self, search):
 
 
 class TestUriFilter(object):
+    def test_filters_by_uri(self, search, Annotation):
+        Annotation(target_uri="http://bar.com")
+        Annotation(target_uri="https://bar.com")
+        Annotation(target_uri="https://foo/bar.com")
+        expected_ids = [Annotation(target_uri="bar.com").id]
+        expected_ids.sort()
+
+        result = search.run({"uri": "bar.com"})
+        result.annotation_ids.sort()
+
+        assert result.annotation_ids == expected_ids
+
+    def test_filters_on_http_and_https_url(self, search, Annotation):
```

Comment:
Is this a feature or a bug? Is http://foo.bar supposed to match https://foo.bar?

Reply:
Good question. This is a feature 😀. Search for mentions of "httpx" in Slack/Github for the backstory.